pkg/rego/load.go

```diff
@@ -1,21 +1,15 @@
 package rego
 
 import (
-	"embed"
 	"io/fs"
 	"os"
 	"path/filepath"
 
+	embedded "github.com/fugue/regula/rego"
```

Comment:
I got rid of the build step that copies the regula library and rules to `pkg/rego`. It simplifies the build and I think it will make it easier to use Regula as a go library.